ops/model.py

```diff
@@ -1445,6 +1445,10 @@ def set_content(self, content: Dict[str, str]):
         secret no longer exists, this method will succeed, but the unit will go
         into error state on completion of the current Juju hook.
 
+        .. jujuchanged:: 3.6
+            Setting the content with the same content as the latest revision
+            longer creates a new secret revision.
+
```

Comment:
Should it be 'no longer creates'?

Rephrasing idea to eliminate redundancy, feel free to disregard
```
            A new secret revision will *not* be created if the content being set
            is identical to the latest revision.
```

Reply:
Done, thank you!